Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/MainPresenter.java

```diff
@@ -111,4 +120,30 @@ public void onDirectorySelected(String dir)
   {
     mDirToAdd = dir;
   }
+
+  public void installWAD(String file)
+  {
+    Thread installWADThread = new Thread(() ->
+    {
+      Looper.prepare();
+      AlertDialog dialog = new AlertDialog.Builder(mContext).create();
```

Comment:
To match the newly merged PR #8718, please use `R.style.DolphinDialogBase` here. (You will need to rebase on latest master first.)